W-22382746-Add-FIPS-140-3-to-Studio-CP (#640)
mulesoft/docs-studio: modules/ROOT/pages/studio-fips-140-3-support.adoc

= Anypoint Studio FIPS 140-3 Environment Support
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> Anypoint Studio can run in environments configured for FIPS 140-3-compliant cryptography. This support helps teams in regulated organizations keep development workflows aligned with security requirements.

Reviewer: ```suggestion
You can now run Anypoint Studio in environments configured for FIPS 140-3-compliant cryptography. This ensures your teams keep development workflows aligned with security requirements.
```

Author: OK! Thanks!

> FIPS support means Studio is compatible with a compliant operating system and Java cryptographic configuration. 

Reviewer: ```suggestion
Support for FIPS makes Studio compatible with compliant operating systems and Java cryptographic configurations. 
```

Author: OK! Thanks!
